README.md

```diff
@@ -1,12 +1,15 @@
 [(Français)](#contr%C3%B4leur-bloc-note-prob)
 
+
 ## Prob Notebook Controller
 
 Based on https://github.com/StatCan/kubeflow-controller
 **There is a dependency on that repository** as well since it needs to be imported in order to use the `Notebook` struct and `NotebookInformer`.
 
-This controller creates and deletes Authorization Policies based on Notebooks. 
-The Authorization Policies block uploads and downloads on jupyterlab and rstudio images.
+This controller creates and deletes Authorization Policies based on Notebooks to insure protected B notebooks have the correct restrictions. 
+Specifically, it watches the creation of Notebook CRDs and creates authorization policies for protected B notebooks. It can also detect if a protected B notebook did not have the policies. 
+
+The Authorization Policies are generated in the handler.go and [blocks specific paths](https://github.com/StatCan/aaw-prob-notebook-controller/blob/dc1cd29e818c413b24385b406f4c22a0bdbf009a/pkg/controller/handler.go#L87-L117) those API calls/paths are the ones used in the uploads and downloads on jupyterlab and rstudio images.
```

Comment:
It would be nice to also see an example of the implementation 

i.e.
```
spec:
  action: DENY
  rules:
    - to:
        - operation:
            methods:
              - POST
            paths:
              - /notebook/namspace-name/test1/rstudio/upload
        - operation:
            methods:
              - GET
            paths:
              - /notebook/namspace-name/test1/rstudio/export*
        - operation:
            methods:
              - GET
            paths:
              - /notebook/namspace-name/test1/files*
        - operation:
            methods:
              - GET
            paths:
              - /notebook/namspace-name/test1/nbconvert*
  selector:
    matchLabels:
      data.statcan.gc.ca/classification: protected-b
      notebook-name: test-pb
```